jenkins/check_file_eol.sh

```diff
@@ -37,4 +37,10 @@ if [ $? -eq 0 ]; then # grep found matches ($?==0), so we found CR (0x0d) in the
     echo "--------------" >> $ERRFILE # same length as '--- ERRORS ---'
 fi
 
+# Check that each file ends in a newline character
+tail -c1 $1 | od -x | grep '000a'
```

Comment:
The previous step redirects output from `grep` to `$ERRFILETEMP`. Do we want to do that here too?